my-app/src/routes/PostDetailPage.jsx

```diff
@@ -1,5 +1,43 @@
+import { useState, useEffect } from "react";
+import { useParams, Link } from "react-router-dom";
+import { BigPost } from "../components/Posts";
+import posts from "../data/posts";
+
 const PostDetailPage = () => {
-  return <div>Post Detail Page</div>;
+  const { postId } = useParams();
+  const [post, setPost] = useState();
```

Comment:
useState 훅을 사용할 때는 초기값을 설정해 두는 게 좋아요!! (가끔 이거 때문에 에러 날 때도 있더라고요...)
```suggestion
  const [post, setPost] = useState(null);
```